packages/react/src/exampleTypes/StepSlider.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { FC, useState } from 'react'
+import _ from 'lodash/fp.js'
+import {
+  RangeSlider,
+  RangeSliderTrack,
+  RangeSliderFilledTrack,
+  RangeSliderThumb,
+  Box,
+  Tooltip,
+  Text,
+  HStack,
+  Stack,
+} from '@chakra-ui/react'
+import { contexturify } from '../utils/hoc'
+import { toNumber } from '../utils/format'
+import { FormatterFn, Node, Tree } from '../types/contexture'
+
+/** This theme is an initial step towards looking further
+ * at the tokenization to be set by our design system. */
+const theme = {
+  color: {
+    blue: { 100: '#CCE7FF', 500: '#0076DE', 700: '#005199' },
+    gray: { 50: '#FFFFFF', 100: '#E7EAEE' },
+    container: { border: { sm: '#E7EAEE' } },
+    text: {
+      body: {
+        primary: '#495569',
+        tertiary: '#697996',
+      },
+    },
+  },
+  radius: { base: { xs: '8px' } },
+  font: { size: { xs: '10px', md: '14px' } },
+}
+
+export type UnknownRec = Record<string, unknown>
+
+type NodeProps = {
+  range: number[]
+  steps: number[]
+}
+
+type ContextProps = {
+  recordsCount: number
+}
+
+interface StepSliderProps {
+  node: Node<NodeProps, ContextProps>
+  formatter: FormatterFn
+  path: string
+  tree: Tree<NodeProps>
+}
+
+const getDefaultRange = (steps: number[], range: number[]): number[] => {
+  if (range.length === 0) return [0, steps.length - 1]
```

Comment:
💎 Suuuper minor, just geeking on pattern for a sec. Thoughts?
```suggestion
  if (!range.length) return [0, steps.length - 1]
```

Reply:
_mmmm_, I like that... (my time to geek on patterns now) but my brain understands `range.length === 0` faster than `!range. length`. For the second I would have to think _137ms_ more to understand "Oooh! When array is empty` 🤣 

But in the end, feel free to choose one.

Reply:
Good point, I'm happy with either, I like the sleekness of `!range.length`, but I appreciate the explicitness of ===0.

Please leave it as is. Thanks for indulging in my geekiness. 😄